src/pages/notification/components/SeenButton/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Button from '~/common/components/Button';
+import useSeenNotification from '~/common/hooks/mutations/useSeenNotification';
+
+const SeenButton = () => {
+  const mutation = useSeenNotification();
```

Comment:
```suggestion
  const { mutate } = useSeenNotification();
```

취향입니다ㅎㅎ 굳이 라고 생각 드신다면 지나치셔도 돼요!